packages/data-fetcher/src/config.spec.ts

```diff
@@ -21,6 +21,9 @@ describe("config", () => {
         rpcCallRetriesMaxTotalTimeout: 120000,
         rpcCallConnectionTimeout: 60000,
         rpcCallConnectionQuickTimeout: 10000,
+        rpcBatchMaxCount: 1,
```

Comment:
Do we have a spec test where this is not 1?

Reply:
Added tests to cover cases when newly added env vars set to a different values.